Makefile

```diff
@@ -56,6 +56,28 @@ ifneq ($(TAG),latest)
 BUNDLE_VERSION ?= --version=$(shell sed s/^v// <<< $(TAG))
 endif
 
+# To get the GIT commit id
+ifndef GIT_COMMIT
+GIT_COMMIT=$(shell git rev-list -1 HEAD)
+endif
+
+# Fetch the git tag
+ifndef GIT_TAG
+GIT_TAG=$(shell git describe --tags --exact-match --abbrev=0 2>/dev/null)
+endif
+
+# In the above command, if it fails to fetch the git tag, then this
+# command will fetch the short commit hash
+ifeq ($(GIT_TAG),)
+GIT_TAG=$(shell git rev-parse --short HEAD)
+endif
+
+GO_PROJECT=github.com/csi-addons/kubernetes-csi-addons
+
+LDFLAGS ?=
+LDFLAGS += -X $(GO_PROJECT)/sidecar/internal/version.GitCommit=$(GIT_COMMIT)
```

Comment:
can you place this file in `internal/version/...` ? That way it can easily be used by both the manager and side-car.

Reply:
added for manager as well.
```
bin/manager -version
Version: v0.6.0
Git Commit: c433e0dac9a8e694e61c363f2e8aff5d3643b3a8
Go Version: go1.19.9
Compiler: gc
Platform: linux/amd64
```